src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -110,7 +110,7 @@ class OrganizationMetricsDataEndpoint(OrganizationEndpoint):
     # XXX: this should be aligned with sessions_v2 (which is SNUBA_LIMIT =
     # 5000), but that triggers another error when querying series, as combined
     # with groupby/rollup, MAX_POINTS can be exceeded easily.
-    default_per_page = 50
+    default_per_page = MAX_POINTS / 2
```

Comment:
Why divide by 2? Is this to be closer to the 5000 of the sessions API? I would just use MAX_POINTS - 1 here.